### PR TITLE
new icon: kdeneon (original, plain)

### LINKED ISSUE #1754: [ICON REQUEST]: KDE Neon
### I have searched through the issues and didn't find my problem.

- [X] Confirm

### About the icon

Icon of the KDE Neon distro

### Links and sources

[KDE Neon Website](https://neon.kde.org)
[Logo Source](https://collaborate.kde.org/s/gT44wHjpDdqqDxL?path=%2F)

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by dotarjun**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR closes #1754**

## Link to prove your SVG is correct and up-to-date.

https://neon.kde.org/

### CODE REVIEW

File: devicon.json [hunk 1]
```diff
@@ -2650,6 +2650,19 @@
...
     },
+    {
+        "name": "kdeneon", 
+        "tags": [
+            "linux",
+            "operating system",
+            "open-source"
+        ], 
+        "versions": {
+            "SVG": ["original"],
```
Comment (Panquesito7) [inline devicon.json:2661]:
```suggestion
            "svg": ["original"], 
```

File: devicon.json [hunk 2]
```diff
@@ -2650,6 +2650,19 @@
...
+        "tags": [
+            "linux",
+            "operating system",
+            "open-source"
+        ], 
+        "versions": {
+            "SVG": ["original"], 
+            "font": ["original"] 
+        },
+        "color": "#21769a"
```
Comment (Panquesito7) [inline devicon.json:2664]:
```suggestion
        "color": "#21769a",
        "aliases": []
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Please create a `plain` version as well to add it to the font versions.

**Comment by dotarjun** [comment]
@Panquesito7  pls check whether the plain svg is good enough

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Please resolve the merge conflicts [**here**](https://github.com/devicons/devicon/pull/1774/conflicts). If you need any help, you can ask for help here or on our [**Discord server**](https://discord.gg/hScy8KWACQ). You can also check the [GitHub Docs](https://docs.github.com/en/github/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github) to make the process easier. 🙂

File: devicon.json [hunk 3]
```diff
@@ -6789,8 +6789,21 @@
...
-        "name": "phalcon",
-        "altnames": [],
+        "name": "kdeneon", 
+        "tags": [
+            "linux",
+            "operating system",
+            "open-source"
+        ], 
+        "versions": {
+            "svg": ["original"],
```
Comment (Panquesito7) [inline devicon.json:6799]:
```suggestion
            "svg": [
                "original",
                "plain"
            ], 
```

File: devicon.json [hunk 4]
```diff
@@ -6789,8 +6789,21 @@
...
-        "altnames": [],
+        "name": "kdeneon", 
+        "tags": [
+            "linux",
+            "operating system",
+            "open-source"
+        ], 
+        "versions": {
+            "svg": ["original"], 
+            "font": ["original"]
```
Comment (Panquesito7) [inline devicon.json:6800]:
```suggestion
            "font": [
                "plain"
            ] 
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by dotarjun — COMMENTED** [review]
I think we're ready to merge it 🤞🏻

File: devicon.json [hunk 5]
```diff
@@ -6789,8 +6789,26 @@
         ]
     },
     {
-        "name": "phalcon",
```
Comment (Panquesito7) [inline devicon.json:6792]: Why was `phalcon` removed?
Or this is a Git diff error? 🤔

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Comment by dotarjun** [comment]
I'm genuinely confused what happened. Closing this PR and working on this issue from a clean slate.

**Pull request closed without merging**